spec/beaker/host_prebuilt_steps_spec.rb

```diff
@@ -516,12 +516,20 @@
     it "can exec the get_ip command" do
       host = make_host('name', { :stdout => "192.168.2.130\n" } )
 
-      expect( Beaker::Command ).to receive( :new ).with( "ip a|awk '/global/{print$2}' | cut -d/ -f1 | head -1", [], {:prepend_cmds=>nil, :cmdexe=>false} ).once
+      expect( Beaker::Command ).to receive( :new ).with( "ip a | awk '/global/{print$2}' | cut -d/ -f1 | head -1", [], {:prepend_cmds=>nil, :cmdexe=>false} ).once
 
       expect( subject.get_ip( host ) ).to be === "192.168.2.130"
 
     end
 
+    it "can exec the get_ip command with tail with vagrant hypervisor" do
+      host = make_host('name', { :stdout => "192.168.2.131\n", :hypervisor => "vagrant" } )
```

Comment:
Since the change you're making is about whether to get the first or last element of the list, I'd think we'd want to use a list that has more than one element.  That way we can detect if it actually grabbed the first or last.